shims/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright (c) 2020, NVIDIA CORPORATION.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.nvidia</groupId>
+        <artifactId>rapids-4-spark-parent</artifactId>
+        <version>0.2.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <groupId>com.nvidia</groupId>
+    <artifactId>rapids-4-spark-shims_aggregator_2.12</artifactId>
+    <packaging>pom</packaging>
+    <name>RAPIDS Accelerator for Apache Spark SQL Plugin Shims</name>
+    <description>The RAPIDS SQL plugin for Apache Spark Shims</description>
+    <version>0.2.0-SNAPSHOT</version>
+
+    <modules>
+        <module>spark30</module>
```

Comment:
Just FYI First and Last changed recently between 3.0.0 and 3.0.1 so they are no longer compatible.  Do we want to name these so it is clearer? perhaps `spark3_0_0`.  Or is there going to be some kind of a hierarchy so a 3.0.1 shim can depend on spark30?

Reply:
So I made it 30 as I wasn't sure what would change in minor release and figured we could create a spark301 if we needed. We can certainly make it 300 right now if you want.  I haven't tested with it to see what all is different there.  There is no hierarchy at this point, I think that could get confusing pretty quickly.

Reply:
I agree that it could get confusing quickly. I am also concerned about a lot of copy/paste.  The changes that went into 3.0.1 for First/Last are the same that went into 3.1. So does that mean we copy/paste these twice? What about for version 3.0.2 when nothing changed between 3.0.1 and 3.0.2 except one minor thing.  Do we have to copy and paste code changes for each minor release from here on out?

Reply:
right now yes, actually I take my first statement, I would prefer to leave this as 30 and 31 until we pull in stuff from 3.0.1 and so forth. We can add modules and stuff ot that and rearrange that that point. My problem now is that it takes a long time to everyday upmerge to spark 3.1, and other rapids changes and things keep moving, then I have to constant retest all versions.  I would like to get a base set of changes in and we can iterate on it.  We could look at making other hierarchies but you have to watch for circular dependencies and such.